frontend/src/components/dialogs/SnapshotImportDialog.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+    <ff-dialog ref="dialog" :header="title" confirm-label="Upload" :disable-primary="!formValid" :closeOnConfirm="false" @confirm="confirm()" @cancel="cancel">
+        <template #default>
+            <form class="space-y-6 mt-2" @submit.prevent>
+                <FormRow :error="validateField.file ? errors.file : ''" data-form="import-snapshot-filename">
+                    Snapshot File
+                    <template #input>
+                        <ff-text-input v-model="input.file" :error="errors.file" disabled />
+                        <!-- <div class="ff-input ff-text-input">
+                            <input type="text" readonly class="bg-gray-50 border border-gray-300  focus:outline-none text-gray-500 cursor-not-allowed" :value="input.file">
+                        </div> -->
+                    </template>
+                    <template #append>
+                        <input id="fileUpload" ref="fileUpload" type="file" accept="application/json, text/plain, *" class="" style="/*position: absolute; top: -9999px;*/ display:none;">
```

Comment:
commented out inline `style`?

Reply:
ah yes, before I figured out how to cypress test a hidden file input.

This is what it should be ↓

```suggestion
                        <input id="fileUpload" ref="fileUpload" type="file" accept="application/json, text/plain, *" style="display:none;">
```